lib/persistence/http/RestClient.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:coffeecard/model/AppConfig.dart';
+import 'package:coffeecard/model/Login.dart';
+import 'package:coffeecard/model/Token.dart';
+import 'package:dio/dio.dart';
+import 'package:retrofit/retrofit.dart';
+
+part 'RestClient.g.dart';
+
+@RestApi()
+abstract class RestClient {
+  static const String PRODUCTION_URL = "https://analogio.dk/clippy";
+  static const String DEBUG_URL = "https://beta.analogio.dk/api/clippy";
```

Comment:
Since we normally refer to this as the test environment, is there any reason why we want to call it debug all of a sudden?

Reply:
That probably be ideal in the long run if we do that instead.

Reply:
I'll create it as a Trello task so we can configure out how to do it properly

Reply:
Trello task https://trello.com/c/qszXRpMk/40-set-restclient-url-in-build-configuration